x-pack/plugins/monitoring/dev_docs/reference/metricbeat.yarn.yml

```diff
@@ -4,6 +4,17 @@ http.enabled: true
 http.host: "0.0.0.0"
 
 metricbeat.modules:
+  - module: system
+  - module: nginx
```

Comment:
Q: This is not part of the telemetry right? it's nice to have it in the docs tho 🙂

Reply:
Ooops. thanks for catching it.